cognite/client/data_classes/filters.py

```diff
@@ -89,20 +89,17 @@ def dump(self, camel_case_property: bool = False) -> dict[str, Any]:
 
     @classmethod
     def load(cls, filter_: dict[str, Any]) -> Filter:
-        (filter_name,) = filter_
-        filter_body = filter_[filter_name]
-
-        if filter_name == And._filter_name:
+        if filter_body := filter_.get(And._filter_name):
```

Comment:
Hmm. I think 'and' and 'or' filters can return empty arrays, or we might allow it at some point in the future. So this needs to be more precise with its truthiness check.